component-classes/index.js

```diff
@@ -509,8 +509,8 @@ export const toaster = {
 
 // TOGGLE COMPONENTS:
 export const switchToggle = {
-  label: 'block relative h-24 w-44 cursor-pointer group',
-  labelDisabled: 'pointer-events-none',
+  base: 'block relative h-24 w-44 cursor-pointer group focusable rounded-full',
```

Comment:
Just a side note, because of how Vue Switch is built, this `focusable` class has no effect when tabbing to it as far as I tested. Fixing the focus outline there would need to be a separate task but perhaps it's better to build a reusable Switch Element instead of rewriting the current Vue Switch.